index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>FCC</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<header id="header">
+  <img id="header-img"
+    src="https://www.google.com/url?sa=i&url=https%3A%2F%2Flogo.com%2F&psig=AOvVaw2-aQ5XPwk3wXX-VffkMwQY&ust=1641223279637000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCIDx7qivk_UCFQAAAAAdAAAAABAD">
+  <nav id="nav-bar">
+    <ul>
+      <li><a class="nav-link" href="#features">Features</a></li>
+      <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
```

Comment:
Add your header inside the body tags

Reply:
- [ ] Great job pointing out the changes required. However, it's best to make it descriptive, polite and friendly, instead of saying Add your header inside the body tag, you can try Please add your header inside the body tag... and give a reason for it. You can check out this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate).

Reply:
Please add the header tag inside of the body tag because the header of a page is part of the body of the page